src/parcsr_ls/par_mgr_setup.c

```diff
@@ -1406,12 +1406,8 @@ hypre_MGRSetupFrelaxVcycleData( void *mgr_vdata,
                         num_functions, dof_func_array[lev_local], CF_marker_local,
                         &coarse_dof_func_lvl, &coarse_pnts_global_lvl);
 
-#ifdef HYPRE_NO_GLOBAL_PARTITION
     if (my_id == (num_procs -1)) coarse_size = coarse_pnts_global_lvl[1];
       hypre_MPI_Bcast(&coarse_size, 1, HYPRE_MPI_BIG_INT, num_procs-1, comm);
```

Comment:
@rfalgout @oseikuffuor1 This is not the focus of this PR. but the indentation of this line (`hypre_MPI_Bcast`) is confusing.

Reply:
@liruipeng I can fix this in PR #260, thanks.